### PR TITLE
feat: Add a "minutely" timeline

### PR DESCRIPTION
**Pull request by jullanggit**
## Disclaimer
I am not very experienced with contributing to projects like rustic, so please correct me if I make any mistakes or could improve something about this PR!
In this case, I am particularly unsure about the commit message and how to add tests, so any tips there would be appreciated :smile:
## Motivation
As creating a rustic snapshot can be really fast if not much has changed (less than 10s), I have started to backup-up my files every ten minutes, but noticed that rustic doesn't yet have any keep-options for sub-hourly snapshots, so I decided to add them!

### CODE REVIEW

File: crates/core/src/commands/forget.rs [hunk 1]
```diff
@@ -356,12 +370,31 @@ fn equal_hour(sn1: &SnapshotFile, sn2: &SnapshotFile) -> bool {
...
+/// Evaluate the minutes of the given snapshots
+///
+/// # Arguments
+///
+/// * `sn1` - The first snapshot
+/// * `sn2` - The second snapshot
+///
+/// # Returns
+///
+/// Whether the hours of the snapshots are equal
```
Comment (aawsome) [inline crates/core/src/commands/forget.rs:382]: should be `minutes` here..

**Comment by simonsan** [comment]
Please also add an explanation to `./config/full.toml` and the `./config/Readme.md` 📓

**Comment by aawsome** [comment]
@jullanggit Ah, I think these are due to a new Rust version. We should fix those in a separate PR before we merge this one...

**Comment by aawsome** [comment]
It would be great if you could extend/adapt the unittests in `forget.rs` to cover the new option, @jullanggit Besides this, this PR looks good to me, just needs to wait until we fix the clippy lints in master. Thanks a lot for the contribution!

**Comment by simonsan** [comment]
The only thing missing now for clippy is the `#[allow(clippy::too_many_lines)]` annotation on `matches()`.

**Review by simonsan — APPROVED** [review]

**Comment by simonsan** [comment]
Thank you for the contribution! 🚀 :)

**Pull request merged**